draft-ietf-rtcweb-jsep.xml

```diff
@@ -1275,38 +1275,41 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
-            and start the media flow. While this could also be done
-            with an inactive "pranswer", followed by a sendrecv
-            "answer", the initial "pranswer" leaves the offer-answer
-            exchange open, which means that neither side can send an
-            updated offer during this time.</t>
-
-            <t>As an example, consider a typical web application that
-            will set up a data channel, an audio channel, and a video
-            channel. When an endpoint receives an offer with these
-            channels, it could send an answer accepting the data
-            channel for two-way data, and accepting the audio and video
-            tracks as inactive or receive-only. It could then ask the
-            user to accept the call, acquire the local media streams,
-            and send a new offer to the remote side moving the audio
-            and video to be two-way media. By the time the human has
-            accepted the call and triggered the new offer, it is likely
-            that the ICE and DTLS handshaking for all the channels will
-            already have finished.</t>
+            <t>Most applications will not need to create answers using the
+            "pranswer" type. While it is good practice to send an immediate
+            response to an offer, in order to warm up the session transport
+            and prevent media clipping, the preferred handling for a JSEP
+            application is to create and send a "sendonly" final answer with a
+            null MediaStreamTrack immediately after receiving the offer, which
+            will prevent media from being sent by the caller, and allow media
+            to be sent immediately upon answer by the callee.
+            Later, when the callee actually accepts the call, the application
+            can plug in the real MediaStreamTrack and create a new "sendrecv"
+            offer to update the previous offer/answer pair and start
+            bidirectional media flow. While this could also be done with a
+            "sendonly" pranswer, followed by a "sendrecv" answer, the initial
+            pranswer leaves the offer-answer exchange open, which means that
+            the caller cannot send an updated offer during this time.</t>
+
+            <t>As an example, consider a typical JSEP application that wants to
+            set up audio and video. When the callee
+            receives an offer with these MediaStreamTracks, it could send an
+            answer accepting the audio and video tracks as sendonly (meaning
+            that the caller will not send the callee any media yet, and because
+            the callee has not yet added its own MediaStreamTracks, the callee
+            will not send any media either). It could
+            then ask the user to accept the call, acquire the needed local
+            tracks, and, upon acceptance, send a new offer to the remote side
```

Comment:
seems you might want one point added in here which is after acceptance, it can add the tracks which will case media to start being sent right away, and then it go on to send 2nd offer.